tests/src/actionContainers/Swift3ActionContainerTests.scala

```diff
@@ -41,6 +41,16 @@ class Swift3ActionContainerTests extends SwiftActionContainerTests {
          |     return ["auth": auth, "edge": edge]
          |}
          """.stripMargin
+    override lazy val errorCode = """
```

Comment:
Its getting less necessary.  Originally, Swift 3 tests were a superset of Swift 2 tests.  Now some of the common tests have diverged because of the changes in the language.